guides/development/hack_on_the_trento.md

```diff
@@ -85,4 +101,20 @@ docker run -v "$PWD":/data --network host ghcr.io/trento-project/photofinish run
 
 Several useful scenario fixtures are available in [./test/fixtures/scenarios](https://github.com/trento-project/web/tree/main/test/fixtures/scenarios), the same ones used in e2e tests.
 
-See also [.photofinish.toml](https://github.com/trento-project/web/blob/main/.photofinish.toml).
\ No newline at end of file
+See also [.photofinish.toml](https://github.com/trento-project/web/blob/main/.photofinish.toml).
+
+## Connect Trento web with [Wanda](https://github.com/trento-project/wanda)
+
+In order to connect the Trento web with Wanda, you must first have a running Wanda server.
+To install Wanda, please follow the instructions provided in the [hack_on_wanda documentation](https://github.com/trento-project/wanda/blob/update_documentation/guides/development/hack_on_wanda.md).
+Once you have a running Wanda server, you can proceed to connect it with Trento web.
+
+The next step is to restart the Trento web server with the following command:
+
+```
+WANDA_URL=http://localhost:4001 MIX_ENV=wanda iex -S mix phx.server
```

Comment:
See https://github.com/trento-project/web/pull/1137, MIX_ENV=wanda should soon no longer be needed (nor useful), maybe we can adapt this?

Reply:
@rtorrero @EMaksy Let's wait to clarify this, because i think it could be still needed somehow: https://github.com/trento-project/web/pull/1137#discussion_r1088771271

Reply:
Okay waiting for the outcome :D